packages/rest/src/responses.ts

```diff
@@ -94,7 +95,11 @@ function processResponseType(
         const allProperties = (p: ModelType): ModelTypeProperty[] => {
           return [...p.properties.values(), ...(p.baseModel ? allProperties(p.baseModel) : [])];
         };
-        if (allProperties(responseModel).some((p) => !isResponseMetadata(p))) {
+        const childMap = getChildModelMap(program);
```

Comment:
If we're keeping this should the openapi emitter use it too and should we expose mapChildModels?

Reply:
Yes, if this make sense to keep this i;'ll update the openapi emitters.

Reply:
This has been replaced by https://github.com/microsoft/cadl/pull/415